examples/p4est_2d_dgsem/elixir_eulergravity_convergence.jl

```diff
@@ -10,7 +10,7 @@ gamma = 2.0
 equations_euler = CompressibleEulerEquations2D(gamma)
 
 polydeg = 3
-solver_euler = DGSEM(polydeg, flux_hll)
+solver_euler = DGSEM(polydeg, FluxHLL(min_max_speed_naive))
```

Comment:
I kept for the Eulergravity stuff the naive version as there is one example for which the Davis one does (at least for the current configuration) crash. Seems to me sufficiently specialized and a good opportunity to show also the more diffusive naive version.

Reply:
👍